starters/nuxt-2/store/README.md

```diff
@@ -0,0 +1,10 @@
+# STORE
```

Comment:
Probably worth replacing some of this with info related to the existing store. Like the lifecycles of actions and things.

Reply:
Agreed.  I think a follow-up README.md task for the entire repo makes sense